src/Processing.js

```diff
@@ -8049,6 +8051,13 @@
           j = i % PConstants.SINCOS_LENGTH;
           p.vertex(centerX + cosLUT[j] * hr, centerY + sinLUT[j] * vr);
         }
+
+        if (mode === PConstants.CHORD)
+        {
+          startingCoordinate = startLUT % PConstants.SINCOS_LENGTH;
+          p.vertex(centerX + cosLUT[startingCoordinate] * hr, centerY + sinLUT[startingCoordinate] * vr);
+        }
+
         p.endShape(PConstants.CLOSE);
         doStroke = savedStroke;
```

Comment:
we can add a conditional control on the `doStroke` here and get the OPEN behaviour out of this patch then, too.